api/src/main/resources/innsending-api.yml

```diff
@@ -1017,6 +1017,167 @@ paths:
+            500:
+              description: Feil i backend
+              content:
+                application/json:
+                  schema:
+                    $ref: "#/components/schemas/RestErrorResponseDto"
+                  example:
+                    message: Noe gikk galt, prøv igjen senere
+                    timestamp: 2023-12-21T09:31:40.549032434
+                    errorCode: somethingFailedTryLater
+          tags:
+            - eksternoppgave
+
+
+  /ekstern/v1/oppgave/{innsendingsId}:
+        delete:
+            summary: Kall for å slette opprettet søknad eller ettersending gitt innsendingsId.
+            description: Dersom funnet, slettes søknaden/ettersendingen.
+            operationId: eksternOppgaveSlett
+
+            parameters:
+              - name: innsendingsId
+                in: path
+                description: identifisering av søknad/ettersending
+                required: true
+                schema:
+                  type: string
+                style: simple
+              - name: Nav-Call-Id
+                in: header
+                description: En ID som identifiserer kallkjeden som dette kallet er en del av.
+                required: false
+                schema:
+                  type: string
+
+            responses:
+              200:
+                description: Slettet søknad/ettersending
+                content:
+                  application/json:
+                    schema:
+                      $ref: "#/components/schemas/BodyStatusResponseDto"
+              401:
+                description: Uautorisert
+                content:
+                  application/json:
+                    schema:
+                      $ref: "#/components/schemas/RestErrorResponseDto"
+                    example:
+                      message: Autentisering feilet
+                      timestamp: 2023-12-21T09:31:40.549032434
+                      errorCode: errorCode.unauthorized
+              404:
+                description: Ingen ettersending funnet
+                content:
+                  application/json:
+                    schema:
+                      $ref: "#/components/schemas/RestErrorResponseDto"
+                    example:
+                      message: Ingen ettersending funnet
+                      timestamp: 2023-12-21T09:31:40.549032434
+                      errorCode: errorCode.resourceNotFound
+              500:
+                description: Feil i backend
+                content:
+                  application/json:
+                    schema:
+                      $ref: "#/components/schemas/RestErrorResponseDto"
+                    example:
+                      message: Noe gikk galt, prøv igjen senere
+                      timestamp: 2023-12-21T09:31:40.549032434
+                      errorCode: somethingFailedTryLater
+            tags:
+              - eksternoppgave
+
+  /ekstern/v1/oppgave/skjema/{skjemanr}:
```

Comment:
GET `/ekstern/v1/oppgaver?skjemanummer=<skjemanr>`

Reply:
skjemanummer i body i stedet for path?

Reply:
Oppdatert url og Flyttet til body